II.pass-tests/07-second-largest/second-largest.js

```diff
@@ -0,0 +1,7 @@
+ function secondLargest(array) {
```

Comment:
the test for this function is not passing. Can we figure out why?
The reason seems to be similar to exercise 5 in `I.write-tests`
So you can try debugging the `array` and check if it is behaving as you'd expect it to

Reply:
checked the code and is passing, I didn't receive any error